frontend/src/Components/UserProfileInvites.tsx

```diff
@@ -70,9 +71,10 @@ export const UserProfileInvites = observer((props: UserProfileInvitesProps) => {
         return;
     };
 
-    const handleRegenerate = async (code: string) => {
+    const handleRegenerate = async (code: string, idx: number | undefined) => {
         try {
             const result = await api.inviteAPI.regenerate(code);
+            setRegeneratedIdx(idx);
```

Comment:
minor bug: If you regenerate the same invite multiple times in a row, the animation is triggered only once.